harness/determined/deploy/local/cluster_utils.py

```diff
@@ -172,6 +176,10 @@ def master_up(
     if initial_user_password is not None:
         master_conf["security"]["initial_user_password"] = initial_user_password
         make_temp_conf = True
+    elif master_conf["security"].get("initial_user_password") is None:
+        generated_user_password = secrets.token_urlsafe(16)
```

Comment:
what chars does this include? can we make sure it's can be picked up with a double click? eg `-` would break that flow. also maybe we printout what they'd use to create a user 

``
det -u admin u create USERNAME --password $GENRATEDPWD`

to make it easier

actually, why not prompt the user for a pwd if it's an interactive shell session? they can confirm to keep/get the autogenerated version or provide a new one

Reply:
Oh, good point -- I'll try extending this flow a little bit. I think it should still print the autogenerated password in case this is run with a closed stdin (i.e. non-interactively) or the user hits Ctrl+C or something, but prompting to change the password immediately would likely be helpful in the majority of cases. Thanks!

Reply:
this would be great to include as additional cases to your e2e test!